docs/contributor/gcp-permissions.md

```diff
@@ -0,0 +1,55 @@
+# Google Cloud Platform Permissions
+
+## Default Principal
+
+```
+compute.globalAddresses.create
```

Comment:
If we already did a layout with nested unordered lists for [Azure](https://github.com/kyma-project/cloud-manager/blob/main/docs/contributor/azure-permissions.md) shouldn't we do it here as well?

Reply:
In my opinion, this one is easier to use (copy-paste).
We just need to make sure values are sorted.